### PR TITLE
fix(imgs): updated images

### PR DESCRIPTION
**Pull request by nwhite222**
Various updates to dark mode

### CODE REVIEW

**Review by rhetoric101 — DISMISSED** [review]
@nwhite222 thanks for updating these! I added a few minor comments privately.

File: src/images/browser_screenshot-full_filtered-web-vitals.webp
Comment (nwhite222) [inline src/images/browser_screenshot-full_filtered-web-vitals.webp:1]: good

File: src/images/accounts_screenshot-full_user-menu-location.webp
Comment (nwhite222) [inline src/images/accounts_screenshot-full_user-menu-location.webp:1]: good

**Review by rhetoric101 — APPROVED** [review]
@nwhite222 thanks for making these improvements.

**Pull request merged**